### ISSUE TITLE
Simulator - Debug Mode

### ISSUE THREAD

**Issue by robthomson** [FrSkyRC/ETHOS-Feedback-Community#4620]
It would be a great addition to the simulator to have a debug mode option that can be enabled.

Will save having to use the CLI to load it!

![debug](https://github.com/user-attachments/assets/d1941201-855d-4af5-888f-17dfaf3d3679)

**Comment by Richrfl** [comment]
Rob, when you reference debug, do you mean for developers or users? This is what I mean: I would like to be able to enter values, for example, Vertical Speed, to activate a logical switch that will generate some action. In my models (all fixed-wing electric) I am using up to 40 LSs to generate from stall warnings to automatic motor reverse and although the LSs have been enhanced a lot, it is still really hard to follow the process when you have a chain of LSs that have to check different telemetry data to produce a final action. As of today, the only way to check that the process I did is correct, is in the field but it should be safer and efficient to simulate the conditions in the Sim and avoid putting models at risk.

**Comment by robthomson** [comment]
I am referring to a debug console for developers to use when coding lua scripts

**Comment by Richrfl** [comment]
Well, we are in different business. Although I did some LUAs, my focus is not on development. Thank you for your response.
